src/patternfly/components/Wizard/docs/code.md

```diff
@@ -0,0 +1,42 @@
+## Accessibility
+
+| Attribute | Applied To | Outcome |
+| -- | -- | -- |
+| `aria-expanded="true"` | `.pf-c-wizard__toggle` | Indicates that the steps menu is visible. **Required** |
+| `aria-expanded="false"` | `.pf-c-wizard__toggle` | Indicates that the steps menu is hidden. **Required** |
+| `aria-label="close"` | `.pf-c-wizard__toggle-icon` | Gives the close button an accessible name. **Required** |
+| `aria-hidden="true"` | `.pf-c-wizard__toggle-icon`, `.pf-c-wizard__toggle-divider` | Hides the icon from assistive technologies. |
+| `aria-label="Steps"` | `.pf-c-wizard__nav` | Gives the steps nav element an accessible name. **Required** |
+| `aria-disabled="true"` | `.pf-c-wizard__nav-link` | Indicatese that the element is disabled. **Required for disabled links with `.pf-m-disabled`** |
+| `aria-current="page"` | `.pf-c-wizard__nav-link` | Indicates the current page link. Can only occur once on page. **Required for the current link** |
+| `tabindex="-1"` | `.pf-c-wizard__nav-link` | Removes a link from keyboard focus. **Required for disabled links with `.pf-m-disabled`** |
+
+## Usage
+
+| Class | Applied To | Outcome |
+| -- | -- | -- |
+| `.pf-c-wizard` | `<div>` | Initiates the wizard component. **Required** |
+| `.pf-c-wizard__header` | `<header>` | Initiates the header . **Required** |
+| `.pf-c-wizard__close` | `.pf-c-button.pf-m-plain` | Initiates the close button. **Required** |
+| `.pf-c-wizard__title` | `.pf-c-title.pf-m-3xl` | Initiates the title. **Required** |
+| `.pf-c-wizard__description` | `<p>` | Initiates the description. |
+| `.pf-c-wizard__steps` | `<div>` | Initiates the steps container. **Required** |
+| `.pf-c-wizard__toggle` | `<button>` | Initiates the mobile steps menu toggle button. **Required** |
+| `.pf-c-wizard__toggle-list` | `<ol>` | Initiates the toggle list. **Required** |
+| `.pf-c-wizard__toggle-list-item` | `<li>` | Initiates a toggle list item. **Required** |
+| `.pf-c-wizard__toggle-num` | `<span>` | Initiates the step number. **Required** |
+| `.pf-c-wizard__toggle-divider` | `<i>` | Initiates the divider between steps. |
+| `.pf-c-wizard__toggle-icon` | `<i>` | Initiates the toggle icon. **Required** |
+| `.pf-c-wizard__nav` | `<nav>` | Initiates the steps nav. **Required** |
+| `.pf-c-wizard__nav-list` | `<ol>` | Initiates a list of steps. **Required** |
+| `.pf-c-wizard__nav-item` | `<li>` | Initiates a step list item. **Required** |
+| `.pf-c-wizard__nav-link` | `<a>` | Initiates a step link. **Required** |
+| `.pf-c-wizard__main` | `<main>` | Initiates the main container. **Required** |
```

Comment:
Could `.pf-c-wizard__main` take on the padding of `.pf-c-wizard__body`. I don't see styles for `.pf-c-wizard__main` so I don't see the need for both of these classes, but let me know if I'm wrong :)

Reply:
I talked to @kybaker and we're treating this similar to a card. With the body element, you get the proper spacing that allows you just to put content in it and it works. Without that element, the main area is just a container where you can put anything you want, and we don't impose padding or anything.

Reply:
Ok that sounds good to me